Assets/HoloToolkit/UX/Scripts/Collections/CollectionEnums/OrientTypeEnum.cs

```diff
@@ -11,7 +11,11 @@ public enum OrientTypeEnum
         None,                   // Don't rotate at all
         FaceOrigin,             // Rotate towards the origin
         FaceOriginReversed,     // Rotate towards the origin + 180 degrees
-        FaceFoward,             // Zero rotation
```

Comment:
Might want to keep these but mark them obsolete to prevent breaking changes.

Be sure to point people to which enum they should be using.

Reply:
Unity serializes enums as integers, so as long as they're in the same position in the enum, the existing data reflects the rename. FaceForward and FaceForwardReversed will transition automatically to FaceParentForward and FaceParentBack.

As for references in code, they will be a compile error, but I hope these are easy enough to find and replace. I think this is a fair trade for getting it out of the inspector. I'll add a comment to the new value explaining its old name.

Reply:
Relevant for upvotes:
https://feedback.unity3d.com/suggestions/enum-members-decorated-with-hideininspector-should-not-appear-in-an-enum-popup

Reply:
Please make this change:

```
        [Obsolete("Please use FaceParentFoward")]
        FaceFoward = FaceParentFoward,             // Zero rotation
        [Obsolete("Please use FaceParentBack")]
        FaceForwardReversed = FaceParentBack,    // Zero rotation + 180 degrees
        FaceParentFoward,       // Parent Relative Forwards
        FaceParentBack,         // Parent Relative Backwards
        FaceParentUp,           // Parent Relative Up
        FaceParentDown,         // Parent Relative Down
        FaceCenterAxis,         // Lay flat on the surface, facing in
        FaceCenterAxisReversed // Lay flat on the surface, facing out
```

Reply:
Hmm. I'm not familiar with setting an enum to another value in the enum, and your sample doesn't compile. Can you elaborate on what you're suggesting?

The compile error I'm getting is:
The evaluation of the constant value for 'OrientTypeEnum.FaceForwardReversed' involves a circular definition

Reply:
Oh, this is because FaceParentBack is defined after FaceForwardReversed. Hold on.

Reply:
This works:
```C#
        FaceParentFoward,       // Parent Relative Forwards, this used to be called FaceForward
        FaceParentBack,         // Parent Relative Backwards, this used to be called FaceForwardReversed
        [Obsolete("Please use FaceParentFoward")]
        FaceFoward = FaceParentFoward,             // Zero rotation
        [Obsolete("Please use FaceParentBack")]
        FaceForwardReversed = FaceParentBack,    // Zero rotation + 180 degrees
        FaceParentUp,           // Parent Relative Up
        FaceParentDown,         // Parent Relative Down
```
But I almost think it's not worth the rename, since it replaces a confusing name in the inspector with an invalid confusing name and a clear name. Unless there's a way to fix that, I'll just put the names back to the old ones.

Reply:
I just wrote it real quick, but I think you get the idea.